bowling/inc/lane.hpp

```diff
@@ -1,8 +1,24 @@
 #pragma once
 
+#include "singleGame.hpp"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
 class Lane {
 private:
+    std::vector<SingleGame> gamesInLine_{};
+    std::string laneName_{};
+
+    void parseFile(std::string fileName);
+    void parseLaneName(std::string laneName);
+
 public:
-    Lane();
+    explicit Lane(std::string fileName);
+    Lane() = delete;
     ~Lane();
+
+    std::string getLaneName();
```

Comment:
such one line getter should be implemented in hpp file directly (inline).

Reply:
I always forget about it, thanks